src/components/Banner/SpringSurgeBanner.style.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+import { COLOR_MAP } from 'common/colors';
+
+export const Wrapper = styled.div`
+  background-color: ${COLOR_MAP.LIGHTGRAY_BG};
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem 1.5rem 1.75rem;
+  align-items: center;
+  justify-content: center;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    flex-direction: row;
+    padding: 2.5rem 1rem;
+  }
+`;
+
+export const InnerContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  a {
```

Comment:
This whole file is copied from VaccinationBanner.style.tsx except I dropped all the button stuff I didn't need and added this styling for links...